docs/root/start/sandboxes/dynamic-configuration-filesystem.rst

```diff
@@ -90,8 +90,8 @@ from ``service1`` to ``service2``:
 .. literalinclude:: _include/dynamic-config-fs/configs/cds.yaml
    :language: yaml
    :linenos:
-   :lines: 7-15
-   :lineno-start: 7
+   :lines: 6-14
+   :lineno-start: 6
```

Comment:
@mattklein123 just to highlight this - if configs in the examples dir are changed we need to check that they arent included in docs with line highlights etc

there arent many of them - they can be found by grepping  for `literalinclude` and `.yaml`